src/server/wtr-config.js

```diff
@@ -215,8 +217,6 @@ export class WTRConfig {
 			});
 		}
 
-		testReporting = !!testReporting || this.#cliArgs['test-reporting'];
```

Comment:
What's the end result going to be for a random repo that didn't previously have this configured? Like there'll now be a JSON file that gets generated when the tests run? Are we worried that it won't be in `.gitignore`?

Reply:
Yeah there will be a `d2l-test-report.json` generated. I don't think it will be a problem for repos. Most repos don't explicitly check in generated files or do anything after running tests.

Reply:
I could add a disable test reporting flag if we think we will need to disable it.

Reply:
Ok cool. I don't think it's worth adding a flag until we know we'll need it.